tools/slang-embed/slang-embed.cpp

```diff
@@ -324,6 +323,23 @@ struct App
         }
     }
 
+    static Slang::UnownedStringSlice getIncludedFileName(Slang::UnownedStringSlice line)
+    {
+        auto trimmedLine = line.trimStart();
+        if (!trimmedLine.startsWith("#"))
+        {
+            return Slang::UnownedStringSlice();
+        }
+        auto preprocessorLine = trimmedLine.tail(sizeof "#" - 1).trimStart();
```

Comment:
The Slang codebase usually uses `sizeof` with parens, e.g. `sizeof("#")`. At least I've never had to think about the precedence of `sizeof` before, not sure if this is confusing to anyone else :smile: